contracts/wawaka/attestation-test/scripts/attestation-test.psh

```diff
@@ -91,4 +99,30 @@ ledger -q -s _sig_ current-state -i ${_id1_} -p signature
 attestation -q -f ${save}/contract1.pdo reveal_secret -s _secret_ -a ${_sig_}
 echo ${HEADER} secret is ${_secret_} ${ENDC}
 
+## =================================================================
+echo ${INFO} test sgx report verification ${ENDC}
+
+set -q -s _report_ -f ${test_data}/report.js
+set -q -s _certificate_ -f ${test_data}/certificate.pem
+set -q -s _signature_ -f ${test_data}/signature.txt
+set -q -s _bad_signature_ -f ${test_data}/bad_signature.txt
+
+attestation -q -f ${save}/contract1.pdo verify_sgx_report -s _r_ -c ${_certificate_} -r ${_report_} -i ${_signature_}
+if -e ${_r_} false
+   echo failed to verify SGX report
+   exit -v 1
+fi
+
+attestation -q -f ${save}/contract1.pdo verify_sgx_report -s _r_ -c ${_certificate_} -r ${_report_} -i ${_bad_signature_}
+if -e ${_r_} true
+   echo failed to identify invalid SGX signature
+   exit -v 1
+fi
+
+attestation -q -f ${save}/contract1.pdo verify_sgx_report -s _r_ -c ${_certificate_} -r "badreport" -i ${_signature_}
+if -e ${_r_} true
+   echo failed to identify invalid SGX report
+   exit -v 1
+fi
```

Comment:
These tests work.
However, they work only because they test one (out of 5) return values of `verify_sgx_report_wrapper`.
The first 3 `return false` (i.e., error) would translate to `true` here -- which is not ok.
The last one -- `return VERIFY_FAILURE`, equivalent to `true` -- should translate to `false` here -- which is ok.